server/auth/types/openid/routes.ts

```diff
@@ -162,21 +174,23 @@ export class OpenIdAuthRoutes {
               authHeaderValue: `Bearer ${tokenResponse.idToken}`,
               expires_at: getExpirationDate(tokenResponse),
             },
-            authType: 'openid',
+            authType: AuthType.OPEN_ID,
             expiryTime: Date.now() + this.config.session.ttl,
           };
+
           if (this.config.openid?.refresh_tokens && tokenResponse.refreshToken) {
             Object.assign(sessionStorage.credentials, {
               refresh_token: tokenResponse.refreshToken,
             });
           }
           this.sessionStorageFactory.asScoped(request).set(sessionStorage);
+
           return response.redirected({
             headers: {
               location: nextUrl,
             },
           });
-        } catch (error) {
+        } catch (error: any) {
```

Comment:
For my knowledge, why are we explicitly declaring the error to be of type `any` here and a bunch of other places?

Reply:
Typescript complains about the attribute type missing. Add any type to avoid the warning.